doc/operator-guide/runbook.rst

```diff
@@ -0,0 +1,5 @@
+########
+RunBook
+########
+
+Maybe from ConsDb Usage Confluence page?
```

Comment:
Actually, most of this should be from (or link to) https://rubinobs.atlassian.net/wiki/spaces/LSSTOps/pages/45665320/Consolidated+Database+ConsDB+Runbook+draft+incomplete (which doesn't have much in it right now either).